src/presentation/controller/registration/Routing.ts

```diff
@@ -28,12 +30,19 @@ const registrationRoutingPartnershipType = {
 export const registrationRoutingName = {
   previousUrl: WHICH_TYPE_URL,
   currentUrl: NAME_URL,
-  nextUrl: NEXT_URL,
+  nextUrl: EMAIL_URL,
   pageType: RegistrationPageType.name,
 };
 
-const registrationRoutingNext = {
+const registrationRoutingEmail = {
   previousUrl: WHICH_TYPE_URL,
+  currentUrl: EMAIL_URL,
```

Comment:
It is intended to return to WHICH_TYPE_URL
because NAME_URL needs the query parameter
so the user has to select an option before going to the name page

Reply:
AC6 on the story says that user is returned to the partnership name page. So that screen is going to have to somehow cope with that. Might be tricky and we'll need another story for that?

Reply:
back link updated 
use browser history